android/app/src/main/res/values/dimens.xml

```diff
@@ -15,8 +15,12 @@
     <dimen name="home_logo_width">200dp</dimen>
 
     <!--card view specifics -->
-    <dimen name="card_elevation">2dp</dimen>
-    <dimen name="card_corner_radius">0dp</dimen>
+    <dimen name="card_small_elevation">2dp</dimen>
+    <dimen name="card_medium_elevation">4dp</dimen>
+    <dimen name="card_large_elevation">6dp</dimen>
+    <dimen name="card_corner_small_radius">2dp</dimen>
+    <dimen name="card_corner_medium_radius">4dp</dimen>
+    <dimen name="card_corner_large_radius">6dp</dimen>
```

Comment:
No need to create so many dimens. I think just fixing the default value is enough

Reply:
Okay 👍